### PR TITLE
[CN-1194] Add warning on expose externally in Istio setup

### PR DESCRIPTION
**Pull request by semihbkgr**

### CODE REVIEW

File: docs/modules/ROOT/pages/connect-outside-kubernetes.adoc [hunk 1]
```diff
@@ -57,6 +57,9 @@ To allow connections from clients, you need to configure the `exposeExternally`
 - `LoadBalancer` (default): Hazelcast client configuration `<load-balancer-external-ip>:5701`.
 +
 WARNING: Your Kubernetes cluster should be able to assign public IPs to LoadBalancer services.
+
+WARNING: Expose externally feature is not supported in Istio setup.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/connect-outside-kubernetes.adoc:61]: Two warnings one after another is not a good look, I have added a comment to combine both warnings, so lines 59 an 60 will need to be removed.

Comment (semihbkgr) [reply docs/modules/ROOT/pages/connect-outside-kubernetes.adoc:61]: ohh, I forgot about that

Comment (rebekah-lawrence) [reply docs/modules/ROOT/pages/connect-outside-kubernetes.adoc:61]: Actually, I think the second warning should be an IMPORTANT admonition, and belongs at the top of the file; so,

Line 42, put a new paragraph after the first sentence, then the IMPORTANT admonition, then another new para for the introduction to the table. Like this:

```
== Configuring `exposeExternally`

To allow connections from clients, you need to configure the `exposeExternally` field, of the Hazelcast custom resource. 

IMPORTANT: If using Itsio, you cannot use `exposeExternally`.

This field has the following properties:
```
Is there something else that they can use for the same effect?

**Review by rebekah-lawrence — COMMENTED** [review]
For your consideration...

File: docs/modules/ROOT/pages/connect-outside-kubernetes.adoc (hunk 1 above)
Comment (semihbkgr) [reply docs/modules/ROOT/pages/connect-outside-kubernetes.adoc:61]: no unfortunately

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

Thanks!

**Pull request merged**